features/keychain/module/keychain.js

```diff
@@ -43,6 +47,32 @@ export class Keychain extends ExodusModule {
     this.secp256k1 = secp256k1.create({ getPrivateHDKey: this.#getPrivateHDKey })
   }
 
+  #assertPrivateKeysUnlocked() {
+    assert(!this.#privateKeysAreLocked, 'private keys are locked')
+  }
+
+  arePrivateKeysLocked() {
+    return this.#privateKeysAreLocked
+  }
+
+  lockPrivateKeys() {
+    this.#privateKeysAreLocked = true
+  }
+
+  unlockPrivateKeys(seed) {
```

Comment:
shouldn't this be an array?

Reply:
Sure, will do that, no downside. There are breaking changes to master anyway so didn't figure it matters & made it simpler